www/scripts/codecheckerviewer/BugFilterView.js

```diff
@@ -1407,6 +1448,28 @@ function (declare, Deferred, dom, domClass, all, topic, Standby, Button,
     },
 
     /**
+     * Deselecting items which are not active by the URL state.
```

Comment:
Do we need this in this pull request? It does nothing right now.

Reply:
Yes, we need this. I didn't have time to write this function last night, I wrote it this morning and I removed the WIP label from this pull request.